src/languages/python.js

```diff
@@ -7,13 +7,19 @@ Category: common
 
 export default function(hljs) {
   var KEYWORDS = {
+    $pattern: /(?<!\.)\w+/,
```

Comment:
Can't use negative look-behinds yet, browser support isn't good enough.

Reply:
> Built-ins aren’t highlighted directly after a dot; t

You might have to look into `beginKeywords` if you insist on solving this - as it includes that support manually. (to not match keywords after `.`)  Or you could use the callbacks, but this currently isn't possible with the build-in `keyword` support.

Reply:
I don’t think it’s very important, let’s skip it for now.